lib/membership/index.js

```diff
@@ -230,6 +236,31 @@ Membership.prototype.makeSuspect = function makeSuspect(address, incarnationNumb
         Member.Status.suspect, this.localMember));
 };
 
+Membership.prototype.makeTombstone = function makeTombstone(address, incarnationNumber) {
+    this.ringpop.stat('increment', 'make-tombstone');
+    return this._updateMember(new Update(address, incarnationNumber,
+        Member.Status.tombstone, this.localMember));
+};
+
+Membership.prototype.removeMember = function removeMember(address) {
+    var hasMember = _.has(this.membersByAddress, address);
+    if (!hasMember) {
+        return;
+    }
+    var memberToDelete = this.membersByAddress[address];
+    delete this.membersByAddress[address];
+    this.members = _.without(this.members, memberToDelete);
+    this.computeChecksum();
+};
+
+Membership.prototype.evict = function evict(address) {
+    if (this.localMember.address === address) {
+        this.logger.error('ringpop tried to evict the local member from the memberlist, action has been prevented');
+        return;
+    }
+    this.removeMember(address);
```

Comment:
For some background: It has been implemented in go in two separate functions because of this discussion: https://github.com/uber/ringpop-go/pull/123#discussion_r55337319

I think it is very beneficial to have the code in go and node as close as possible to each other so it is easy to port features from 1 language to the other.